docs/networking.md

```diff
@@ -545,7 +545,7 @@ You can specify which subnets to use for allocating Pod IPs by specifying
   networking:
     lyftvpc:
       subnetTags:
-        kubernetes_kubelet: true
+        KubernetesCluster: myclustername.mydns.io
```

Comment:
Sorry one more thing. the KubernetesCluster tag is now specified by default and can't be removed. So maybe the docs should make that clear?

> Subnets to be used for Pod IP allocation need to be tagged with `KubernetesCluster: myclustername.mydns.io`. Additional tags can be specified to further filter which subnets are used:
> 
> ```
>  networking:
>    lyftvpc:
>      subnetTags:
>        PodIPs: true
> ```

Reply:
I think it can be removed and add any other tag that makes sense. This is more of a power user CNI plugin. Anyone messing with the tags will have to understand the consequences.

Reply:
ah you're right, the `tags` variable is reassigned if the `subnetTags` field is specified, so it is possible to override the default KubernetesCluster tag. 👍